docs/ADMIN.md

```diff
@@ -73,6 +73,10 @@ On Windows, the easiest way to retain your logs is to use [redirection](https://
 
 Advanced users should consider sending their logs to a log aggregator such as [Grafana Cloud](https://grafana.com/products/cloud/logs/). If you do this, I also recommend using `--log-format=json` to log in JSON format, which is easier to search and filter when using an aggregator.
 
+## Config Path
```

Comment:
I'm planning to update this guide and the powershell script/cloud-init/systemd unit to use config files, as they have a few advantages:

1. They're less intimidating for non-technical users than editing a script
2. They don't expose passwords in the process tree or environment - yeah, DCS/SRS passwords are rarely secret, but it will make a difference if anything important like Discord API access is ever needed

I'll do that update pass before the next release